munyurangeri/ghost-dev-exercise
# Dynamic content and animation

## 📃 Summary
Use json-server to serve dummy content, cache the content using service worker and indexedDB. It also adds some animations. 


## 👀 Steps to QA

1. Run `npm install`
2. Run `npm run preview` 
3. Now if you click on "Next Client Story" you will get a random story and if it has some pictures, they will be displayed in slider otherwise a default place holder picture will be displayed.
4. Open Dev Tools and go to **Lighthouse** and check if all metrics are green.
5. Open Dev Tools and go to **Application** and check if there are contents in both **Cash storage and IndexedDB**



## 🎰 Risk Factors
None



## ✅ Checklist






## 🎆 Screenshots
